examples/Handler Methods.ipynb

```diff
@@ -0,0 +1,717 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Handler Methods\n",
+    "\n",
+    "<p style=\"font-size:18px\">Handler methods define the skeleton of the execution algorithm. It let's steps override some part of the algorithm without changing the structure. It makes it possible to build really powerful steps that can edit, and change the execution flow. You can find a detailed BaseStep API reference here.</p>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img src=\"images/handler_methods.png\" style=\"width=:50\" />"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following handle methods are available for each step: \n",
+    "\n",
+    "\n",
+    "### handle_fit_transform\n",
+    "\n",
+    "1. _will_process(data_container, context): Apply side effects before any step method.\n",
+    "2. _will_fit_transform(data_container, context): Apply side effects before fit_transform\n",
+    "3. _fit_transform_data_container(data_container, context): Fit transform data container.\n",
+    "4. _did_fit_transform(data_container, context): Apply side effects after fit_transform.\n",
+    "5. _did_process(data_container, context): Apply side effects after any step method.\n",
+    "\n",
+    "### handle_fit\n",
+    "\n",
+    "1. _will_process(data_container, context): Apply side effects before any step method.\n",
+    "2. _will_fit_transform(data_container, context): Apply side effects before fit.\n",
+    "3. _fit_data_container(data_container, context): Fit data container.\n",
+    "4. _did_fit(data_container, context): Apply side effects after fit.\n",
+    "\n",
+    "### handle_transform\n",
+    "\n",
+    "1. _will_process(data_container, context): Apply side effects before any step method.\n",
+    "2. _will_transform(data_container, context): Apply side effects before transform.\n",
+    "3. _transform_data_container(data_container, context): Fit transform data container.\n",
+    "4. _did_transform(data_container, context): Apply side effects after transform. \n",
+    "5. _did_process(data_container, context): Apply side effects after any step method.\n",
+    "\n",
+    "### When to use handler methods ? \n",
+    "\n",
+    "When you need to apply side effects, or change the execution flow:\n",
+    "\n",
+    "- Edit the DataContainer\n",
+    "- Call a method on a step\n",
+    "- Mini-Batching\n",
+    "- Caching\n",
+    "- etc.\n",
+    "\n",
+    "### HandleOnlyMixin\n",
```

Comment:
Please [link to the class](https://www.neuraxle.org/stable/api/neuraxle.base.html#neuraxle.base.ForceHandleMixin) as such, using markdown code in your notebook. 

This is a comment in general here, there could be other links to other documentation pages in the present notebook as well. Let's aim for approx 5 links or more to other places in the doc :)